projects/media-viewer/src/lib/media-viewer/toolbar/search-bar/search-bar.component.ts

```diff
@@ -21,6 +22,16 @@ export class SearchBarComponent {
 
   constructor() {}
 
+  @HostListener('window:keydown', ['$event'])
+  onWindowKeyDown(e: KeyboardEvent): void {
+    if (e.code === 'F3' || (e.ctrlKey && e.code === 'KeyF')) {
+      e.preventDefault();
+
+      this.searchBarHidden.next(false);
+      setTimeout(() => this.findInput.nativeElement.focus(), 200);
```

Comment:
does this unfocus the search bar after 200s? does it focus on the window itself afterwards? is unfocussing it standard UI procedure?

Reply:
this focuses it so when you press ctrl+f / f3 the window pops up and you can start typing in the field.

The 200 is milliseconds